update aliyun-terway/usage.md (#4816)
DaoCloud/DaoCloud-docs: docs/en/docs/network/modules/aliyun-terway/usage.md

## Create ECS Instances

For detailed instructions, please refer to the Alibaba Cloud documentation, and keep in mind the following points:

- Choose a region closer to your location to reduce network latency and improve access speed.
- If no VPC exists, create a VPC and select an available zone. This VPC will be used for node and Pod communication.
- Select the appropriate instance specifications based on your requirements. These specifications determine the number of ENIs and secondary IPs available on the ECS instances, which affects the number of Pods that can be run (in non-VPC mode).

![create-ecs](https://docs.daocloud.io/daocloud-docs-images/docs/en/docs/network/images/ECS-Create.png)

!!! note

    Installing a Kubernetes cluster requires each machine to have a minimum of 2 CPUs and 2 GB of memory.

## Set up the Kubernetes Cluster

Refer to the [official documentation](https://kubernetes.io/docs/setup/production-environment/tools/kubeadm/install-kubeadm/) to install kubeadm and kubectl on the control plane node. Install kubelet and a container runtime (e.g., containerd) on both the control plane and worker nodes.

!!! note

>     Modify the kubelet configuration file on each node (`/usr/lib/systemd/system/kubelet.service.d/10-kubeadm.conf`) and configure `--provider-id`:

Reviewer: ```suggestion
    Modify the kubelet configuration file (`/usr/lib/systemd/system/kubelet.service.d/10-kubeadm.conf`) and configure `--provider-id` on each node:
```

2. Install the CCM component with the manifests provided in [cloud-controller-manager.yaml](../../yamls/cloud-controller-manager.yaml).

> !!! note
> 
>     It requires to modify `cluster_cidr` to match your actual podSubnet(10.244.0.0/16).

Reviewer: ```suggestion
    !!! note

        It requires to modify `cluster_cidr` to match your actual podSubnet(10.244.0.0/16).
```

EOF
```

!!! note

>     Declare 'aliyun/eni: 1' in the resources configuration to make the Pod exclusively use the ENI network card.

Reviewer: ```suggestion
    Declare `aliyun/eni: 1` in the resources configuration to make the Pod exclusively use the ENI network card.
```
